### PR TITLE
Design work and about sections

### PR DESCRIPTION
**Pull request by NduatiKagiri**
In this branch I:
Added the work and about sections in HTML file
Added the work and about designs in css file
Added image files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+    <section id="works">
+      <h5>My Recent Works</h5>
+      <div class="works">
+        <div class="stories flex">
+          <div class="stories-image">
+            <img src="assets/stories.png" alt="">
+          </div>
+          <div class="stories-info flex">
+            <div class="stories-title">
+              <p>Multi-Post Stories</p>
```
Comment (AnselemOdims) [inline index.html:73]: - [ ] Please make sure this and subsequent project title use a header tag `h2-h6` for semantic reasons

File: index.html [hunk 2]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+          <div class="grid-head">
+            <h6>Profesional Art<br>Printing Data More</h6>
+          </div>
+          <div class="grid-desc">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          </div>
+          <div class="grid-tools">
+            <ul>
+              <li>
+                <div class="">
```
Comment (AnselemOdims) [inline index.html:176]: - [ ] [OPTIONAL] It is not good practice to have empty classes. Kindly remove it completely if it has no value

File: index.html [hunk 3]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+      <div class="hor-line">
+      </div>
+      <div class="about-tools flex">
+        <div class="languages tool-box">
+          <div class="tools-title">
+            <div class="">
+              <img src="assets/Rectangle1.png" alt="">
+            </div>
+            <div class="">
+              <h6>Languages</h6>
```
Comment (AnselemOdims) [inline index.html:274]: - [ ] Please kindly ensure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

File: index.html [hunk 4]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+                </li>
+                <li>
+                  <div class="flex">
+                    <p>ruby</p>
+                  </div>
+                </li>
+              </ul>
+            </div>
+            <div class="stories-see">
+              <button class="flex" type="button" name="button">See Project</button>
```
Comment (AnselemOdims) [inline index.html:103]: - [ ] The list of technologies and the `button` are not properly aligned like that of the original design. Please kindly adjust to meet design requirements

### Your Design
image

### Original Design
image

File: index.html [hunk 5]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+            <ul>
+              <li>
+                <div class="">
+                  <p>html</p>
+                </div>
+              </li>
+              <li>
+                <div class="">
+                  <p>bootstrap</p>
+                </div>
```
Comment (AnselemOdims) [inline index.html:129]: - [ ] Please ensure that your design for the project cards meet the original design guidelines(fonts, margins, padding, etc). At the moment, the buttons are not appearing on each card as specified in the original design

### Your Design
image

### Original Design
image

File: index.html [hunk 6]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+        <div class="about-info flex">
+          <div class="info-head">
+            <h6>About me</h6>
+          </div>
+          <div class="info-desc">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          </div>
+          <div class="info-btn">
+            <button class="flex" type="button" name="button">Get My Resume</button>
+          </div>
```
Comment (AnselemOdims) [inline index.html:262]: - [ ] At the moment, the about section does not align with the original design guidelines in terms of spacing, fonts and colors. Please kindly compare your design with the original design and make necessary changes
- [ ] Also, the background image is not visible. Please kindly add the background image using the CSS `background-image` property

### Your Design 
image

### Original Design
image

File: index.html [hunk 7]
```diff
@@ -60,5 +60,262 @@ <h2 class="flex">I’am a Software Developer</h2>
...
+          <div class="grid-desc">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          </div>
+          <div class="grid-tools">
+            <ul>
+              <li>
+                <div class="">
+                  <p>html</p>
+                </div>
+              </li>
```
Comment (AnselemOdims) [inline index.html:152]: - [ ] Please remove the empty whitespace to the right of the page that causes horizontal scroll.

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @NduatiKagiri, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @NduatiKagiri👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**